src/playwright/merge-coverage-reports/create-coverage-reports.ts

```diff
@@ -0,0 +1,70 @@
+import path from 'node:path';
+import { type ReportNode, type Context } from 'istanbul-lib-report';
+import { create, type LinkMapper } from 'istanbul-reports';
+import { coverageReporterOutputDirectory, reporters } from './configuration.js';
+
+export default (context: Context) => {
+  for (const [name, options] of reporters) {
+    if (['text', 'text-summary'].includes(name)) {
+      // Both "text" and "text-summary" are logged to the console. So we put some
+      // vertical space between them to make the output look a little nicer.
+      //
+      // eslint-disable-next-line no-console
+      console.log('\n');
+    }
+
+    if (name === 'html') {
+      // eslint-disable-next-line no-console
+      console.log(`- Writing "html" report to ${context.dir}`);
```

Comment:
Another super nit, but the above file uses single quotes:


```suggestion
      console.log(`- Writing 'html' report to ${context.dir}`);
```

Reply:
You don't think it makes sense to use double quotes given it's an American English reference to the name of a thing?